docs/dev-guide/contributing.md

```diff
@@ -185,4 +214,17 @@ npx webpack-bundle-analyzer build/app-stats.json
 
 ### Kotlin
 
-For Kotlin code we use the [standard convention](https://kotlinlang.org/docs/coding-conventions.html) and limit line length to 120 characters. We use `ktlint` to enforce formatting.
+- For Kotlin code we use the [standard convention](https://kotlinlang.org/docs/coding-conventions.html) and limit line length to 120 characters. We use `ktlint` to enforce formatting.
+
+## ✅ Code Reviews
+
+- **Submit Your Contribution:** After completing your work, submit your contribution.
+- **Automatic Assignment:** Your pull request will be automatically assigned to appropriate reviewers based on the affected codebase and expertise required for the changes.
```

Comment:
I don't think we have that setup. Unless it's something automagical from github?